tests/lib/rules/no-unused-prop-types.js

```diff
@@ -1801,34 +1801,6 @@ ruleTester.run('no-unused-prop-types', rule, {
         '  bar: PropTypes.bool',
         '};'
       ].join('\n')
-    }, {
```

Comment:
So these tests are now able to detect that `lastname` is not used. I understand this is a breaking change? Not insisting on any of these to go in, but seems like a good change to me.